corehq/apps/app_manager/templates/app_manager/v2/app_view.html

```diff
@@ -61,39 +61,50 @@
   {% endif %}
 {% endblock %}
 
-{% block form-view %}
-  <div class="pull-right">
-    <a class="btn btn-default js-preview-toggle" style="display: none;" href="#">
-      <span class="js-preview-action-show hide"><i class="fa fa-play-circle"></i> {% trans "Preview Changes" %}</span>
-      <span class="js-preview-action-hide hide"><i class="fa fa-pause-circle"></i> {% trans "Hide Preview" %}</span>
-    </a>
-    <a href="{% url "app_summary" domain app.id %}" class="btn btn-default">
-      <i class="fa fa-list-alt"></i>
-      {% trans "App Summary" %}
-    </a>
+{% block pre_page_content %}
+
+  <div class="appmanger-page-actions">
```

Comment:
fix spelling (looks like this is spelled consistently within the PR, so not a bug, just for the sake of sanity)